sql/migrations/20200610185313_world.sql

```diff
@@ -0,0 +1,81 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200610185313');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200610185313');
+-- Add your query below.
+
+
+-- Pattern: Icy Cloak - Micha Yance
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 2381 AND `item` = 4355;
+
+-- Formula: Enchant Bracer - Lesser Deflection - Micha Yance
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 43200 WHERE `entry` = 2381 AND `item` = 11163;
+
+-- Weighted Throwing Axe - Sarah Raycroft
+DELETE FROM `npc_vendor` WHERE `entry` = 3541 AND `item` = 3131;
+
+-- Balanced Throwing Dagger - Sarah Raycroft
+DELETE FROM `npc_vendor` WHERE `entry` = 3541 AND `item` = 2946;
+
+-- Striped Yellowtail - Hal McAllister
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 3540 AND `item` = 21552;
+
+-- Strong Fishing Pole - Lindea Rabonne
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 2383 AND `item` = 6365;
+
+-- Aquadynamic Fish Attractor - Lindea Rabonne
+UPDATE `npc_vendor` SET `maxcount` = 1 WHERE `entry` = 2383 AND `item` = 6533;

```

Comment:
On wowhead it shows the max count as 2.
https://classic.wowhead.com/npc=2383/lindea-rabonne